pkg/cmd/repo/repo.go

```diff
@@ -23,6 +24,7 @@ func NewCmdRepo(f *cmdutil.Factory) *cobra.Command {
 			$ gh repo create
 			$ gh repo clone cli/cli
 			$ gh repo view --web
+			$ gh repo deploy-key list
```

Comment:
Let's remove this example. Not everyone uses deploy keys and we don't have to necessarily show off every `gh repo` subcommand in these examples.